api/capi/include/nntrainer.h

```diff
@@ -611,6 +611,23 @@ int ml_train_model_save(ml_train_model_h model, const char *file_path,
 int ml_train_model_load(ml_train_model_h model, const char *file_path,
                         ml_train_model_format_e format);
 
+/**
+ * @brief Get neural network layer from the model with the given name.
+ * @details Use this function to get already created Neural Network Layer. The
+ * returned layer must not be deleted as it is owned by the model.
+ * @since_tizen 6.x
+ * @remark The modification through ml_trin_layer_set_property() after compiling
+ * the model by calling `ml_train_model_compile()` strictly restricted.
+ * @param[in] model The NNTrainer model handler from the given description.
+ * @param[in] layer_name Name of the already created layer.
+ * @param[out] layer The NNTrainer Layer handler from the given description.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful.
+ * @retval #ML_ERROR_INVALID_PARAMETER Invalid parameter.
```

Comment:
If ml_train_model_get_layer() comes to be a public function, it should be able to return _ERROR_NOT_SUPPORTED when the related feature is not supported on the device, (ml_train_*** API requires tizen.org/feature/machine_learning and tizen.org/feature/machine_learning as related features.)

Reply:
Platform feature keys confirmed:  http://tizen.org/feature/machine_learning , http://tizen.org/feature/machine_learning.training

Reply:
fixed.